front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -460,7 +466,7 @@ function ActionCard({
             action.description ? "text-element-700" : "text-warning-500"
           )}
         >
-          {action.description || "Missing description"}
+          {action.description || "Missing description. Click to edit."}
```

Comment:
Should we we say "Legacy tool, click to update it." ?

Reply:
Ed delivered some designs, just implemented it 🙇🏻‍♀️